lib/Dialect/Substrait/IR/Substrait.cpp

```diff
@@ -358,6 +369,35 @@ void printDecimalNumber(AsmPrinter &printer, DecimalType type,
   printer << "P = " << type.getPrecision() << ", S = " << type.getScale();
 }
 
+ParseResult parseFixedBinaryTypeRemoveLength(AsmParser &parser,
```

Comment:
Maybe just `FixedBinary` or `FixedBinaryLiteral` is enough? Then we have "a custom assembly for the fixed binary literal" and "`(print|parse)FixedBinaryLiteral` prints|parses the fixed binary literal"?